#!/usr/bin/env python
PACKAGE = "smarc_bt"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_alt", double_t, 0, "Minimum Altitude", default=3, min=-1, max=50)
gen.add("max_depth", double_t, 0, "Maximum Depth", default=15, min=0, max=50)
gen.add("enable_missions", bool_t, 0, "Enable Missions", default=True)


# 3rd one must be the same as filename w/o extension for reasons
exit(gen.generate(PACKAGE, "smarc_bt_cfg", "smarc_bt"))


